### PR TITLE
NF: timber uses format

### PR DESCRIPTION
**Pull request by Arthur-Milchior**

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/ReviewerTest.java [hunk 1]
```diff
@@ -68,7 +68,7 @@
     public void setUp() {
         super.setUp();
         try {
-            Timber.d("scheduler version is " + schedVersion);
+            Timber.d("scheduler version is %s", schedVersion);
```
Comment (david-allison-1) [inline AnkiDroid/src/test/java/com/ichi2/anki/ReviewerTest.java:71]:
```suggestion
            Timber.d("scheduler version is %d", schedVersion);
```

**Review by david-allison-1 — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java [hunk 2]
```diff
@@ -1686,9 +1686,9 @@ private void writeLog(Card newCard, double outcomeProb) {
             String tabs = "";
             for(int d = 0; d<nPrevRevs; d++)
                 tabs += "\t";
-            Timber.d(tabs + "t=" + tElapsed + " p=" + prob + " * " + outcomeProb);
-            Timber.d(tabs + prevCard);
-            Timber.d(tabs + newCard);
+            Timber.d("%st=%d p=%d * %s", tabs, tElapsed, prob, outcomeProb);
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java:1689]: prob is a double

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Was this done via automated refactoring suggestions? 

Some of these appear incorrect

File: AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java (hunk 2 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java:1689]: I automated what I could, i.e. the one of the form `"string" + var`
I did the end myself and checked that it compiles, I thought that I would get warning if this was not properly done, as with usual formatting rules

Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java:1689]: Somehow, I did not realize that double uses %f, and not %d

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java [hunk 3]
```diff
@@ -1262,7 +1262,7 @@ protected void generateQuestionSoundList() {
 
     protected void lookUpOrSelectText() {
         if (clipboardHasText()) {
-            Timber.d("Clipboard has text = " + clipboardHasText());
+            Timber.d("Clipboard has text = %s", clipboardHasText());
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:1265]: bool

File: AnkiDroid/src/main/java/com/ichi2/anki/DeckOptions.java [hunk 4]
```diff
@@ -199,7 +199,7 @@ public boolean commit() {
                     for (Entry<String, Object> entry : mUpdate.valueSet()) {
                         String key = entry.getKey();
                         Object value = entry.getValue();
-                        Timber.i("Change value for key '" + key + "': " + value);
+                        Timber.i("Change value for key ' %s': %s", key, value);
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/DeckOptions.java:202]:
```suggestion
                        Timber.i("Change value for key '%s': %s", key, value);
```

File: AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java (hunk 2 above)
Comment (david-allison-1) [reply AnkiDroid/src/main/java/com/ichi2/libanki/stats/AdvancedStatistics.java:1689]: Still using a %d

**Review by david-allison-1 — CHANGES_REQUESTED** [review]

**Review by david-allison-1 — APPROVED** [review]

**Pull request merged**